client/Makefile

```diff
@@ -15,8 +17,6 @@ GO_PACKAGES_REPO_PATH = $(addprefix $(repo_path)/,$(GO_PACKAGES))
 GOFMT = gofmt -e -l -s
 GOTEST = go test --cover --race -v
 
-VERSION := $(shell git describe --tags --abbrev=0 2>/dev/null)+$(shell git rev-parse --short HEAD)
```

Comment:
Okay, I'll leave the two VERSION... versions. :)  Was there a preference to maintaining the previous client VERSION above or can we transition client's to just the `$(shell git rev-parse --short HEAD)` as used by other components?